### PR TITLE
Add document to install ebtables, ethtool

### PR DESCRIPTION
**Pull request by netroby**
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.8 Features: set Milestone to `1.8` and Base Branch to `release-1.8`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.
>
> Please delete this note before submitting the pull request.

Allow edits from maintainers checkbox

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4800)

### CODE REVIEW

File: docs/setup/independent/install-kubeadm.md [hunk 1]
```diff
@@ -51,6 +51,20 @@ The pod network plugin you use (see below) may also require certain ports to be
 open. Since this differs with each pod network plugin, please see the
 documentation for the plugins about what port(s) those need.
 
+## Installing ebtables ethtool
+
+if you see error warning while run ```kubeadm init```
```
Comment (tengqm) [inline docs/setup/independent/install-kubeadm.md]: 1) if -> If
2) delete 'error'
3 run -> running

File: docs/setup/independent/install-kubeadm.md [hunk 2]
```diff
@@ -51,6 +51,20 @@ The pod network plugin you use (see below) may also require certain ports to be
...
+
+```
+[preflight] WARNING: ebtables not found in system path                          
+[preflight] WARNING: ethtool not found in system path                           
+
+```
+You may missing these tools on your linux , please try to installed them .
```
Comment (tengqm) [inline docs/setup/independent/install-kubeadm.md]: 1) may -> may be
2) "linux ," extra space 
3) installed -> install
4) them . extra space

File: docs/setup/independent/install-kubeadm.md [hunk 3]
```diff
@@ -51,6 +51,20 @@ The pod network plugin you use (see below) may also require certain ports to be
...
+
+For ubuntu/debian user, try ```apt install ebtables ethtool```
```
Comment (tengqm) [inline docs/setup/independent/install-kubeadm.md]: user -> users,
triple ``` can be single ?

File: docs/setup/independent/install-kubeadm.md [hunk 4]
```diff
@@ -51,6 +51,24 @@ The pod network plugin you use (see below) may also require certain ports to be
 open. Since this differs with each pod network plugin, please see the
 documentation for the plugins about what port(s) those need.
 
+## Installing ebtables ethtool
+
+If you see warning while running ```kubeadm init```
```
Comment (chenopis) [inline docs/setup/independent/install-kubeadm.md]: suggested edit: "If you see the following warnings while running \`kubeadm init\`"

File: docs/setup/independent/install-kubeadm.md [hunk 5]
```diff
@@ -51,6 +51,24 @@ The pod network plugin you use (see below) may also require certain ports to be
...
+
+```
+[preflight] WARNING: ebtables not found in system path                          
+[preflight] WARNING: ethtool not found in system path                           
+
```
Comment (chenopis) [inline docs/setup/independent/install-kubeadm.md]: This newline should go after the triple back ticks and before the sentence.

File: docs/setup/independent/install-kubeadm.md [hunk 6]
```diff
@@ -51,6 +51,24 @@ The pod network plugin you use (see below) may also require certain ports to be
...
+```
+You may be missing these tools on your linux, please try to install them.
```
Comment (chenopis) [inline docs/setup/independent/install-kubeadm.md]: suggested edit: "Then you may be missing *ebtables* and *ethtool* on your Linux machine. You can install them with the following commands:"

**Comment by netroby** [comment]
Thanks. @chenopis 

Edit as you suggest

**Comment by netroby** [comment]
ping @chenopis 
Hello, can you have a look? any other problem?

**Comment by chenopis** [comment]
@netroby Sorry for the delay!

**Pull request merged**